README.md

```diff
@@ -89,27 +89,27 @@ There are two more methods in this [guide provided by Preact](https://preactjs.c
 
 ## Scale
-When talking about scale for the frontend you endup talking about how many engineers can work on the project without losing their minds with merge conflicts. This framework also likes to call itself progressive since you can easily scale the abtraction level high or as low as possible with a simple script tag.
+When talking about scale for the frontend you end up talking about how many engineers can work on the project without losing their minds with merge conflicts. This framework also likes to call itself progressive since you can easily scale the abtraction level high or as low as possible with a simple script tag.
 
 ## Why
-Vue allows you to use both JSX and Template but more of the time you will find yourself using the template syntax. Unlike React where you're always faced with decision fatigue, in Vue.js you are able to focus on your application code with a clear API that doesn't try to introduce any new terms. We think Vue.js is a great choice for migrating from React and more so for green field projects. If you want to get up and running with your application in minutes you really should give Vue a try.
+Vue allows you to use both JSX and Template but most of the time you will find yourself using the template syntax. Vue also benefits from having a clear API for building applications so you can focus on writing application code rather than the decision fatigue often associated with React projects. We think Vue.js is a great choice for migrating from React, and espcially for green field projects. If you want to get up and running with your application in minutes you really should give Vue a try.
 
 Resources:
 * [Switching From React To Vue.js](https://vuejsdevelopers.com/2017/05/28/switch-from-react-to-vue-js/)
 * [(Vue) Comparison with Other Frameworks](https://vuejs.org/v2/guide/comparison.html)
 
 # React to Angular
-<img width="815" alt="screen shot 2017-08-25 at 2 38 34 pm" src="https://user-images.githubusercontent.com/1016365/29733693-442bba1a-89a3-11e7-95de-4cafc89974eb.png">
+<img width="815" alt="React to Angular" src="https://user-images.githubusercontent.com/1016365/29733693-442bba1a-89a3-11e7-95de-4cafc89974eb.png">
 
-When migrating to Angular from React there are many differences that you should be aware of. While some of the concepts of React remain with Angular, they are different architecturally. Here's a quick list of differences.
+While many React concepts can be found in Angular, you should be aware of quite a few architectutral differences from React to Angular. Here's a quick list of differences.
```

Comment:
@gdi2290 @Mayho
there are quite a few architectural differences from React to Angular that you should be aware of.